pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -10,75 +10,55 @@ enterprise: false
+**NOTE:** You must be a registered user and logged on to the support portal to download this product. New customers must contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
 
-<!-- TBD when 1.2 goes GA:
-# Release Summary
-
-Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated and cost management, across multiple clusters, whether they are a new Konvoy cluster or existing 3rd party/DIY distribution.
--->
+# Release Summary 
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
 
 # Supported Versions
-
 | Kubernetes Support | Version |
 | ------------------ | ------- |
-| **Minimum**        | 1.16.0  |
-| **Maximum**        | 1.19.x  |
-| **Default**        | 1.18.10  |
-
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
-
-## Features/Improvements
-
-- Improved catalog handling
-- Improved Access Control for editing/deleting actions
-- Added a global Infrastructure Provider view
-
-## Bug Fixes
-
-- Fixed an issue where logout wasn't deleting any cookies
-- Smaller UX Bugs and Improvements
-
-## Component Versions
-
-- Addon: `1.2.0-9`
-- Chart: `0.11.3`
-- auto-provisioning (yakcl): `0.4.5`
-- kommaner-federation (yakcl): `0.4.5`
-- kommander-licensing (yakcl): `0.4.5`
-- UI: `6.22.0`
-- kommander-karma: `0.3.10`
-- kubeaddons-catalog: `0.1.12`
-- kommander-thanos: `0.1.15`
-- kubecost: `0.1.12`
-- grafana: `4.6.3`
-
-# Kommander 1.2.0 Beta 0 (August 27th, 2020)
-
-## Features/Improvements
-
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+# New Features and Capabilities 
+
+## Air gapped environments
+Kommander now supports installing and running in an air gapped environment, either on premises or in the cloud. Air gapping is ideal for instances where high-security is a must or where Internet connectivity is either undesirable or unavailable. For more details on setting up Kommander in an air gapped environment, see [Install Kommander air gapped][install_airgapped].
+
+## Component versions
+- Addon: 1.2.0-26
+- Chart: 0.12.3
+- kommander-federation (yakcl): 0.6.2
+- kommander-licensing (yakcl): 0.6.2
+- UI: 6.37.0
+- kommander-karma: 0.3.10
```

Comment:
for these parent charts (kommander-karma, kommander-thanos, kubecost) do we want to also/instead include the actual component versions (karma, thanos, kubecost) which is probably the more useful information?

Reply:
good question, i vote for actual version (appversion?) but i leave it to yall to decide

Reply:
hm, main idea behind these component was to guide the user (us) to the next thing to check. if we'd have `karma 1.2.3` here, we'd not know which chart version got included and it makes it harder to debug IMHO.

that said, maybe having both would be helpful? not sure how deep we should go tho, I feel like having dep->dep->dep is kinda useless at some point.

Reply:
I also prefer the most current version of all components, not just those that have been updated. We try to do this with DCOS as well, and wnat to do that for Konvoy. Consistency would be best.

Reply:
ooh, I like the idea of having both. list all of them and figure out a way to format the updated ones differently?

Reply:
Could we do this for Konvoy too? Or is it too late?

Reply:
Couldn't we do it for Konvoy? I feel like it's not too late. Can always add to it/improve the docs.

Reply:
App versions in case we want to include them:
```
- karma: 0.70
- thanos: 0.10.1
- cost-analyzer: 1.69.0
- grafana: 6.6.0
```